src/py/flwr/server/driver/grpc_driver.py

```diff
@@ -48,11 +48,18 @@
 
 from .driver import Driver
 
-ERROR_MESSAGE_DRIVER_NOT_CONNECTED = """
-[flwr-serverapp] Error: Not connected.
+ERROR_MESSAGE_PUSH_MESSAGES_RESOURCE_EXHAUSTED = """
+[Driver.push_messages] gRPC error occurred:
 
-Call `connect()` on the `GrpcDriverStub` instance before calling any of the other
-`GrpcDriverStub` methods.
+The 2GB gRPC limit has been reached. Consider reducing the number of messages pushed
+in the batch.
```

Comment:
@chongshenng should we add a more detailed error message here? And also for `pull_messages` below?

```suggestion
The 2GB gRPC limit has been reached. Consider reducing the number of messages pushed
at once, or push messages individually:

> msgs = [msg1, msg2, msg3]
> msg_ids = []
> for msg in msgs:
>     msg_id = driver.push_messages([msg])
>     msg_ids += msg_id
```

Reply:
Updated.